contrib/fwupd.spec.in

```diff
@@ -350,6 +350,7 @@ rm ${RPM_BUILD_ROOT}%{_sbindir}/flashrom
 %endif
 %{_libdir}/fwupd-plugins-3/libfu_plugin_unifying.so
 %{_libdir}/fwupd-plugins-3/libfu_plugin_upower.so
+%{_libdir}/fwupd-plugins-3/libfu_plugin_vli_usbhub.so
```

Comment:
What does vli stand for?  Shouldn't this be `via-usbhub`?

Reply:
VLI seems to be some kind of parent company name, of  which VIA is one subsidiary. VIA requested the plugin was named like this.